services/blockchain-indexer/shared/dataService/business/dynamicReward/defaultReward.js

```diff
@@ -0,0 +1,38 @@
+/*
+ * LiskHQ/lisk-service
+ * Copyright © 2022 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ *
+ */
+const { requestConnector } = require('../../../utils/request');
+
+const { getRewardTokenID } = require('./constants');
+
+const getDefaultRewardAtHeight = async (params) => {
+	const response = {
+		data: {},
+		meta: {},
+	};
+	const defaultReward = await requestConnector('getDefaultRewardAtHeight', { height: params.height });
+	const rewardTokenID = await getRewardTokenID();
+
+	response.data = {
+		defaultReward: defaultReward.error ? null : defaultReward,
```

Comment:
I think, we can verify the response for `getDefaultRewardAtHeight`here: https://github.com/LiskHQ/lisk-sdk/blob/development/framework/src/modules/reward/schemas.ts#L56
```suggestion
		defaultReward: defaultReward.error ? null : defaultReward.reward,
```

Reply:
We can directly assign `defaultReward.reward` here, since the `requestConnector` has been now configured to throw error if the response contains error. Updated the suggestion accordingly.

Reply:
Applied. Thanks for the suggestion guys 🙏